Update README.md

adelewg opened the pull request:
Added the name of the authors and also update the WHY section

## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Screenshots (if applicable, else remove this line / section)


## Checklist:
<!--- Put an `x` in all the boxes that apply ! -->
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

adelewg committed:
Update README.md

Added the name of the authors and also update the WHY section

adelewg committed:
Update README.md

Fixed markup on the url to add the angled brackets

koitoror reviewed (changes requested):
use the recommended branch naming conventions
refer to this on conventions -> https://docs.google.com/presentation/d/1z43OH3hGMwhPQ1-learc3tyw0Kh-cjW8nSQmJz9f1wA/edit#slide=id.g8005fdbe5f_0_98

adelewg committed:
update to README.md

feat: added information about the team members
Names of the team with links to their github repositories
